components/client/src/test/integration/scala/com/hotels/styx/client/RetryHandlingSpec.scala

```diff
@@ -1,18 +1,18 @@
 /**
- * Copyright (C) 2013-2018 Expedia Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+  * Copyright (C) 2013-2018 Expedia Inc.
```

Comment:
Is this caused by the formatter? We want to make sure it matches what the copyright notice generator adds or it will add it again.